docs/kss/public/less/_homepage.css

```diff
@@ -0,0 +1,218 @@
+/*Styling for the new home page*/
+/* We are using IDs in order to avoid breaking the standard styling */
+header.hyHomeHeaderMain {
+  /*font sizes per psd-not following the styleguide*/
+  background: url("/images/home/banner_bg.png") top center repeat-x;
+  margin-top: -20px;
+  margin-left: -15px;
+  margin-right: -15px;
+  height: 440px;
+}
+header.hyHomeHeaderMinor {
+  background-color: #f2f3f5;
+  margin-left: -16px;
+  margin-right: -15px;
+}
+.hyHomePage {
+  /*font sizes per psd-not following the styleguide*/
+}
+.hyHomePage h1 {
+  font-size: 35px;
+}
+.hyHomePage h3 {
+  font-size: 40px;
+}
+.hyHomePage .hyHomeHeader p,
+.hyHomePage .hyHomeHeader a {
+  font-size: 18px;
+  font-family: 'Dosis';
+}
+.hyHomePage .hyHomeHeader {
+  text-weight: lighter;
+}
+.hyHomePage .hyHomeHeader2 p {
+  margin-bottom: 0;
+}
+.hyHomePage .hyHomeHeader2 p small {
+  font-size: 14px;
+}
+.hyHomePage .hyHomeHeader {
+  color: white;
+  padding-bottom: 100px;
+  padding-top: 30px;
+}
+@media (min-width: 769px) {
+  .hyHomePage .hyHomeHeader p {
+    padding: 0 0 25px 0;
+  }
+  .hyHomePage .hyHomeHeader .techne-logo {
+    margin: 30px 0;
+  }
+}
+.hyHomePage .hyHomeHeader2 p {
+  line-height: 40px;
+}
+@media (max-width: 769px) {
+  .hyHomePage .hyHomeHeader2 p {
+    line-height: 20px;
+  }
+}
+.hyHomePage .hyHomeHeader3 {
+  margin-bottom: 50px;
+}
+.hyHomePage .hyHomeHeader3,
+.hyHomePage .personaIcons {
+  background-color: #ffffff;
+}
+.hyHomePage .hyHomeHeader3 img,
+.hyHomePage .personaIcons img {
+  margin-bottom: 20px;
+}
+.hyHomePage .hyHomeHeader3 a,
+.hyHomePage .personaIcons a {
+  display: inline-block;
+  cursor: pointer;
+}
+.hyHomePage .hyHomeHeader3 a.hyPersonaText:hover,
+.hyHomePage .personaIcons a.hyPersonaText:hover {
+  text-decoration: none;
+}
+.hyHomePage .hyHomeHeader3 a.hyPersonaText p,
+.hyHomePage .personaIcons a.hyPersonaText p {
+  color: black;
+}
+.hyHomePage .hyHomeContentCentered p {
+  font-size: 16px;
+}
+.hyHomePage p.hyPersonaTitle {
+  font-size: 27px;
+  text-transform: uppercase;
+  color: #000;
+}
+.hyHomePage .hyPersonaTab li {
```

Comment:
use the same rule for all 3 of these definitions below. 

We don't need the same rule defined 3 times